pkg/testdata/models/apis/ecr/0000-00-00/generator-with-late-initialize.yaml

```diff
@@ -0,0 +1,15 @@
+resources:
+  Repository:
+    fields:
+      Name:
+        late_initialize: {}
+      ImageTagMutability:
+        late_initialize:
+          min_backoff_seconds: 5
```

Comment:
I'm still concerned about the maintenance costs we're imposing on service teams that own the controller repos here. They will now have to manually specify pretty much every single optional field as at least `late_initialize: {}`. I understand the concerns about defaulting behavior of server-side override values, though. Maybe there's some compromise we can make in a future enhancement to the code-generator where we have a "default late_initialize value" that can be specified in a generator config file for a particular resource... more thoughts on this would be good (in a future PR, not saying we need to change this PR)

Reply:
Fair enough.